src/fortranapex/apexsh.f90

```diff
@@ -1,130 +1,130 @@
-!******************************************************************************
+! ******************************************************************************
 !
-!  File Name: apexsh.f90
-!  Authors: John Emmert, Art Richmond
-!  Date: 11/13/2009
-!  Version: 1.0
-!  Description: Converts geodetic coordinates to and from Quasi-Dipole and
-!               Modified Apex Coordinates.
+! File Name: apexsh.f90
+! Authors: John Emmert, Art Richmond
+! Date: 11/13/2009
+! Version: 1.0
+! Description: Converts geodetic coordinates to and from Quasi-Dipole and
+! Modified Apex Coordinates.
 !
-!  References: Richmond, A. D., Ionospheric Electrodynamics Using Magnetic Apex
-!                Coordinates, J. Geomag. Geoelectr., 47, 191-212, 1995.
-!              Emmert, J. T., A. D. Richmond, and D. P. Drob, A computationally
-!                compact representation of Magnetic-Apex and Quasi-Dipole
-!                coordinates with smooth base vectors, J. Geophys. Res., 115,
-!                Axxxxx, doi:10.1029/2010JA015326, 2010.
+! References: Richmond, A. D., Ionospheric Electrodynamics Using Magnetic Apex
+! Coordinates, J. Geomag. Geoelectr., 47, 191-212, 1995.
+! Emmert, J. T., A. D. Richmond, and D. P. Drob, A computationally
+! compact representation of Magnetic-Apex and Quasi-Dipole
+! coordinates with smooth base vectors, J. Geophys. Res., 115,
+! Axxxxx, doi:10.1029/2010JA015326, 2010.
```

Comment:
The "page" for this citation needs to be updated

Reply:
Article number [A08322](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2010JA015326).  This has been corrected.